packages/__docs__/src/Document/index.tsx

```diff
@@ -33,6 +33,7 @@ import { SourceCodeEditor } from '@instructure/ui-source-code-editor'
 import { withStyle, jsx } from '@instructure/emotion'
 
 import generateStyle from './styles'
+import functionalComponentThemes from './functionalComponentThemes.ts'
```

Comment:
I get a TS error in this line because the path has an extension

Reply:
Fixed